axiom/orgs.go

```diff
@@ -205,6 +197,14 @@ type OrganizationCreateUpdateRequest struct {
 	Name string `json:"name"`
 }
 
+type wrappedOrganization struct {
+	Organization
+
+	// HINT(lukasmalkmus) This is some future stuff we don't yet support in this
+	// package so we just ignore it for now.
+	ExternalPlan interface{} `json:"externalPlan,omitempty"`
```

Comment:
```suggestion
	_ interface{} `json:"externalPlan,omitempty"`
```
maybe? ExternalPlan isn't used anywhere so it kind of feels like this whole structure isn't needed right now